Makefile

```diff
@@ -0,0 +1,28 @@
+# Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+BINARY_NAME=demo
+
+all: test build
```

Comment:
As you don't store vendored dependencies in your repository, it's good if you ensure our local machines have them. Please use `dep ensure` before each `build`/`run`/`test` target.